### PR TITLE
Feat: working RSA-PSI demo

### PR DESCRIPTION
**Pull request by s0l0ist**
TODO Implement serialization of objects for interoperability across languages.

### CODE REVIEW

**Comment by iamtrask** [comment]
Looking good!

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true
+}
```
Comment (youben11) [inline .vscode/settings.json:4]: You think you should push vscode settings?

File: examples/rsa.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  exponent: 0x10001
+  // Optionally, provide serialized key(s) / key parameters
+  // so we don't have to generate them again.
+  // privateKey: null
+  // publicKey: null
+})
+console.timeEnd('Generating keys')
+
+// Transport the publicKey to the client. Can also save the keys.
+const { publicKey } = server.keys
```
Comment (youben11) [inline examples/rsa.js:28]: Is this js magic or server.keys is returning the publicKey only?

Comment (s0l0ist) [reply examples/rsa.js:28]: `server.keys` returns an object containing both public/private keys. In this specific case, we only want the publicKey so we're using destructuring assignment.

File: .vscode/settings.json (hunk 1 above)
Comment (s0l0ist) [reply .vscode/settings.json:4]: good catch, i'll take out.

Comment (s0l0ist) [reply .vscode/settings.json:4]: Looking back, definitely not. Removing

File: examples/rsa.js (hunk 2 above)
Comment (youben11) [reply examples/rsa.js:28]: JS magic then lol

**Pull request merged**